src/olympia/addons/templates/addons/persona_preview.html

```diff
@@ -25,7 +25,7 @@
             {# L10n: {0} is the number of users. #}
             {{ ngettext("<strong>{0}</strong> user",
                         "<strong>{0}</strong> users",
-                        addon.persona.popularity)|f(addon.persona.popularity|numberfmt)|safe }}
+                        addon.persona.popularity)|fe(addon.persona.popularity|numberfmt)|safe }}
```

Comment:
Actually my point was that it's applied to the whole thing and *not* just the output of `|numberfmt`:
Simplified, the code is:
`{{ ngettext("...", arg)|fe(arg|numberfmt)|safe }}`

If `|safe| was applied only to the output of |numberfmt, it'd be inside the parenthesis, like this:
`{{ ngettext("...", arg)|fe(arg|numberfmt|safe) }}`

But yeah, that's why I think it's redundant. It's applied to the output of the whole `ngettext()|fe` result that has been marked as safe already because it was formatted with `|fe`.